### DISCUSSION TITLE
480p -> 4K upscale speed > 2 causing jerking frames

### DISCUSSION THREAD

**Discussion by voidastro4** [mpv-player/mpv#14582] [General]
Ryzen 5000 U-series, amdgpu
On Sway. 4K screen, scale factor 2.

vo=gpu-next
gpu-api=vulkan
gpu-context=waylandvk
hwdec=vulkan
profile=gpu-hq

With profile=gpu-hq and with or without hwdec (ctrl+h toggle) when a 480p video is played in fullscreen and playback speed is set to something like 2x the video is unwatchable, not only are there dropped frames but the frames seem to be traveling back in time every second.

Is this supposed to happen? Or are you supposed to not be using profile=gpu-hq even on modern hardware?

For one particular video, 1.5x is butter smooth and on 1.75x unwatchable.

In windowed mode even 2.5x is fine, but as the window size crosses a threshold which appears to be 1440p the jerking begins.

**Comment by llyyr** [comment]
It means your hardware is too weak for `profile=high-quality` (gpu-hq was renamed in later versions) at 2x speed. Nothing much mpv can do here

**Comment by haasn** [comment]
Can you share a screen recording of the issue? I don't think frames should ever be traveling "back in time" no matter what hardware.

**Reply by llyyr** [reply]
Sounds like https://github.com/mpv-player/mpv/issues/13811 given that it's vulkan too

**Reply by voidastro4** [reply]
> Sounds like #13811 given that it's vulkan too

It's exactly like this.
@haasn

System load has nothing to do with it for me though.

Once the window size crosses a certain threshold (or is fullscreen) the issue manifests (looks exactly like the demo video in that issue).
And we are talking about a measly 480p video h264. I think it's a bug with profile=high-quality
